ENGAGEHF/Education/Views/VideoPlayers/VideoView.swift

```diff
@@ -0,0 +1,62 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziViews
+import SwiftUI
+
+
+struct VideoView: View {
+    let video: Video
+    
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 20) {
+            VideoPlayer(youtubeId: video.youtubeId)
+                .padding(.top)
```

Comment:
I would suggest to package this in a rounded rectangle (maybe a small border that is using the accent color) that also has a padding to the side to make stand out a bit more on the detail page. I think this would also make it more understandable how the progress indicator is related to the video.